requirements.txt

```diff
@@ -3,6 +3,6 @@ geopy>=1.14.0
 docopt
 pyyaml
 certifi
-hbmqtt
-websockets<=8.1
+amqtt
+websockets<10.0
```

Comment:
Is there a reason to limit it to `<10.0`? At least `amqtt` sets `>=9.0`.
If you would like to prevent issues in the future, may I suggest `<11.0` instead? There are already libs in Home Assistant that require `> 10.0`.

Of course that depends on compatibility foremost.
https://websockets.readthedocs.io/en/stable/project/changelog.html#id2